egs2/TEMPLATE/tts1/tts.sh

```diff
@@ -651,10 +673,17 @@ if [ ${stage} -le 7 ] && [ ${stop_stage} -ge 7 ]; then
         # shellcheck disable=SC2086
         utils/split_scp.pl "${key_file}" ${split_scps}
 
-        # 2. Submit decoding jobs
+        # 2. Check extra options
+        # NOTE(kan-bayashi): Using opts causes duplicated option errors
+        _ex_opts=
+        if "${_use_speech}"; then
+            _ex_opts+="--allow_variable_data_keys true "
```

Comment:
https://github.com/espnet/espnet/blob/4168d53a9bef5aa3b529c1555a9e4fb423afc438/espnet2/tasks/tts.py#L198-L204

We can add `speech` to this list.

(BTW, possibly I'll  change the system of `--data_path_and_name_and_type`... This style seems too flexible than necessary and it's hard to understand)